### PR TITLE
Added Authentication

### PR DESCRIPTION
**Pull request by rwplyler**
Added things nessicary for authentication

### CODE REVIEW

File: AdminToolAPI/AuthenticationHelpers/CustomAuthenticationHandler.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            var claims = new List<Claim>();
+            respObj.permissions.ForEach(per =>
+            {
+                claims.Add(new Claim(per, "true"));
+            });
+            // context.User.Claims.Append();
+            // var identity = new ClaimsIdentity(claims, "basic");
+            // User = new ClaimsPrincipal(identity);
+
+            // var claims = new[] { new Claim("token", token) };
```
Comment (clarson217) [inline AdminToolAPI/AuthenticationHelpers/CustomAuthenticationHandler.cs]: Please remove commented out code.

File: AdminToolAPI/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("AdminTools")]
+        [Authorize]
+        public async Task<ActionResult<string>> GetExample()
+        {
+            return Ok(new { response = "success" });
+        }
+    }
+}
```
Comment (clarson217) [inline AdminToolAPI/Controllers/WeatherForecastController.cs]: Please remove Weather Controller

**Review by clarson217 — CHANGES_REQUESTED** [review]
Please adjust commented sections.

File: AdminToolAPI/AuthenticationHelpers/CustomAuthenticationHandler.cs (hunk 1 above)
Comment (rwplyler) [reply AdminToolAPI/AuthenticationHelpers/CustomAuthenticationHandler.cs]: Didn't notice that was there. It is now removed

**Pull request closed without merging**